helm/botkube/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: botkube
 home: https://www.botkube.io
-version: v0.12.3
+version: v0.12.4
```

Comment:
This change is not required, v0.12.3 is the latest release

Reply:
@dennybaa the version will be bumped automatically at the time of release. So when we release the next version of the helm chart, it will have the correct version.

Reply:
@dennybaa could you please resolve the conflict?